jsonschema-core.xml

```diff
@@ -1587,206 +1587,75 @@
+                                new base URI determined according to "$recursiveAnchor" producing
+                                the final resolved reference URI.
                             </t>
                             <t>
-                                The value of "$recursiveAnchor" MUST be of type boolean, and
-                                MUST be true.  The value false is reserved for possible future use.
+                                Note that in the absence of "$recursiveAnchor" (and in some cases
+                                when it is present", "$recursiveRef"'s behavior is identical to
+                                that of "$ref".
                             </t>
                         </section>
-                        <section title='Dynamically recursive references with "$recursiveRef"'>
+                        <section title='Enabling Recursion with "$recursiveAnchor"'>
                             <t>
-                                The "$recursiveRef" keyword behaves identically to "$ref", except
-                                that if the referenced schema has "$recursiveAnchor" set to true,
-                                then the implementation MUST examine the dynamic scope for the
-                                outermost (first seen) schema document with "$recursiveAnchor"
-                                set to true.  If such a schema document exists, then the target
-                                of the "$recursiveRef" MUST be set to that document's URI, in
-                                place of the URI produced by the rules for "$ref".
+                                The value of the "$recursiveAnchor" property MUST be a boolean.
                             </t>
                             <t>
-                                Note that if the schema referenced by "$recursiveRef" does not
-                                contain "$recursiveAnchor" set to true, or if there are no other
-                                "$recursiveAnchor" keywords set to true anywhere further back in
-                                the dynamic scope, then "$recursiveRef"'s behavior is identical
-                                to that of "$ref".
+                                "$recursiveAnchor" is used to dynamically identify a base URI
+                                at runtime for "$recursiveRef" by marking where such a calculation
+                                can start, and where it stops.  This keyword MUST NOT affect the
+                                base URI of other keywords, unless they are explicitly defined
+                                to rely on it.
+                            </t>
+                            <t>
+                                If set to true, then when the containing schema object is used
+                                as a dynamic reference target, a new base URI is determined
```

Comment:
"as a dynamic reference target (by use of `$ref`)"
Is this what you mean by that phrase?

Reply:
Not quite, but I changed the wording so it should be more clear now.